specs/Checkbox.md

```diff
@@ -173,7 +147,24 @@ Removing the following two props because the ARIA spec dictates role='checkbox'
 
 Props being removed:
 
-ariaPoisitionInSet and ariaSetSize - when writing parent component, user should set these on the checkbox.
+ariaPoisitionInSet and ariaSetSize - when writing parent component, user should set these on the checkbox. ARIA spec dictates role='checkbox' doesn't need aria-posinset and aria-setsize. These are only valid for role='option' which is only in the case the checkbox is a part of a listbox, which is not something we need to account for in the base component API. If the user does need to provide these two props, slotProps could be used to apply additional props to any slot. 
+
+Note: rtl, styles, and theme come from compose or the ThemeProvider. And name has been added to support checkbox in form scenarios.
+
+### Work items and tracking progress on conversion from the current Fluent (Stardust) checkbox to the new Fluent checkbox we're building with this spec: 
+- [ ] Remove `animation` prop.
```

Comment:
A lot of the attributes to remove or change are generic ones - animation, accessibility, handlers, variables, design... - I think it would be good to create a generic proposal document that describes how we want to handle these use cases in the future. That would also be a good place for discussion around such decisions. Then we can apply the conclusions on individual components.

Reply:
I think it's very important for getting consistency across our components, as well. Any conformance tests we could write against such a document would be invaluable, I think.

Reply:
I think @levithomason said he's going to turn this doc with the summary of all the decisions on design & patterns we made so far into a more guiding principles doc: https://hackmd.io/8WievoYRQpiAXbP_rBAecA